README.md

```diff
@@ -1,8 +1,18 @@
 [![Build and Test](https://github.com/dafny-lang/dafny/workflows/Build%20and%20Test/badge.svg)](https://github.com/dafny-lang/dafny/actions?query=workflow%3A%22Build+and+Test%22) [![Gitter](https://badges.gitter.im/dafny-lang/community.svg)](https://gitter.im/dafny-lang/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+Dafny is a **programming language teamed up with a verifier**. As you type in your program, the verifier constantly looks over your shoulder, flags any errors, shows you counterexamples, and congratulates you when your code matches your specifications. When you're done, Dafny can **compile your code to C#, Java, JavaScript or Go** (more to come!), so it can integrate with your existing workflow.
+
+![vs-code-dafny-2 0 0-demo](https://user-images.githubusercontent.com/3601079/140799975-f3ac0925-10d9-4c14-b1a9-cd449854c6ae.gif)
+
+Dafny will give you **assurance that your code meets the specifications you write**, while letting you write both code and specifications in the Dafny programming language itself. Since verification is an integral part of development, it will thus **improve your confidence of avoiding costly late-stage bugs** that are typically missed by testing.
```

Comment:
Saying something like "reduce the risk of costly late-stage bugs" feels slightly better to me. I'm not sure how to phrase it with "confidence" in a way that doesn't sound awkward.

Reply:
Let me use "reduce the risk...". I already talk about assurance above, and confidence is a synonym. Besides, confidence is extrinsic (meaning, a feeling), while reducing risk is intrinsic (it belongs to the code), so that would be even a better value there.